compiler/src/dotty/tools/dotc/core/Phases.scala

```diff
@@ -287,6 +287,10 @@ object Phases {
 
     def isRunnable(implicit ctx: Context): Boolean =
       !ctx.reporter.hasErrors
+        // TODO: This might test an unintended condition.
```

Comment:
I think this doesn't make a difference in practice since this method is never called from a context with a nested reporter, but we could change it to errorsReported for consistency.